### PR TITLE
Brings test coverage up to 100%

### PR DESCRIPTION
**Pull request by reverentgeek**
* removes unneeded brand parameter from images query]
* fs.readdirSync does not return '.' or '..'
* refactors server setup so server config test coverage is 100%
* ignores .vscode settings
* adds testing around handling products
* moves test.key
* machine testing 100%
* adds 100% coverage to random name generator
* adds 100% coverage to firewallRules
* adds 100% test coverage to formatters
* removes products

### CODE REVIEW

File: lib/handlers/products.js [hunk 1]
```diff
@@ -1,77 +1,7 @@
 'use strict';
```
Comment (geek) [inline lib/handlers/products.js]: We can remove products from the cloudapi, this was mainly added before we created the central navigation

File: lib/setup.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const graphqlHandler = function (route, options) {
+  Assert(typeof options.method === 'function', 'method must be a function');
+
+  return function (request, h) {
+    const fetch = request.plugins.cloudapi.fetch.bind(request.plugins.cloudapi);
+    return options.method(fetch, request.payload, request);
+  };
+};
+
+module.exports = {
```
Comment (geek) [inline lib/setup.js]: we should just export directly on each of the functions instead of an export at the bottom.

File: test/firewallRules.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ const Lab = require('lab');
 const StandIn = require('stand-in');
 const CloudApi = require('webconsole-cloudapi-client');
 const ServerHelper = require('./helpers/server');
+const TestDouble = require('testdouble');
```
Comment (geek) [inline test/firewallRules.js]: could alpha sort these with 

node core
3rd party
local

**Review by geek — CHANGES_REQUESTED** [review]

File: lib/setup.js (hunk 2 above)
Comment (reverentgeek) [reply lib/setup.js]: Why is this preferred? Just curious.

**Comment by reverentgeek** [comment]
I made the requested changes, but maybe I shouldn't have squashed them?

**Review by geek — APPROVED** [review]

**Pull request merged**